classes/class-uagb-admin-helper.php

```diff
@@ -409,6 +409,34 @@ public static function get_global_content_width() {
 
 			return '' === $content_width ? 1140 : $content_width;
 		}
+
+		/**
+		 * Function to check if the current theme is a block theme.
+		 *
+		 * @since 2.7.11
+		 * @return boolean
+		 */
+		public static function is_block_theme() {
+			return ( function_exists( 'wp_is_block_theme' ) && wp_is_block_theme() ) ? true : false;
+		}
+
+		/**
+		 * Get Spectra Pro URL with required params
+		 *
+		 * @return string
+		 * @since 2.7.11
```

Comment:
Quick Fix - can be committed from this comment.

```suggestion
		 * @since 2.7.11
		 * @return string
```

Reply:
Done in [this commit](https://github.com/brainstormforce/ultimate-addons-for-gutenberg/pull/3509/commits/a1a41df1f7d7565015bfce0d4d2b47b7690eaf96).

Reply:
Resolved in https://github.com/brainstormforce/ultimate-addons-for-gutenberg/pull/3511